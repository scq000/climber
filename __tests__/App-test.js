import 'react-native';
import React from 'react';
import {create, act} from 'react-test-renderer';
import App from '../App';

it('App', async () => {
  const snap = create(<App />).toJSON();
  await act(async () => {
    expect(snap).toMatchSnapshot();
  });
});
