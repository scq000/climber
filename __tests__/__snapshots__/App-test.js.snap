// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <RNCSafeAreaProvider
    onInsetsChange={[Function]}
    style={
      Array [
        Object {
          "flex": 1,
        },
        undefined,
      ]
    }
  >
    <View
      onLayout={[Function]}
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View
        pointerEvents="box-none"
        style={
          Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <View
          collapsable={false}
          style={
            Object {
              "opacity": 1,
            }
          }
        />
        <View
          accessibilityElementsHidden={false}
          closing={false}
          gestureVelocityImpact={0.3}
          importantForAccessibility="auto"
          onClose={[Function]}
          onGestureBegin={[Function]}
          onGestureCanceled={[Function]}
          onGestureEnd={[Function]}
          onOpen={[Function]}
          onTransitionStart={[Function]}
          pointerEvents="box-none"
          style={
            Object {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
          transitionSpec={
            Object {
              "close": Object {
                "animation": "spring",
                "config": Object {
                  "damping": 500,
                  "mass": 3,
                  "overshootClamping": true,
                  "restDisplacementThreshold": 10,
                  "restSpeedThreshold": 10,
                  "stiffness": 1000,
                },
              },
              "open": Object {
                "animation": "spring",
                "config": Object {
                  "damping": 500,
                  "mass": 3,
                  "overshootClamping": true,
                  "restDisplacementThreshold": 10,
                  "restSpeedThreshold": 10,
                  "stiffness": 1000,
                },
              },
            }
          }
        >
          <View
            needsOffscreenAlphaCompositing={false}
            pointerEvents="box-none"
            style={
              Object {
                "flex": 1,
                "marginTop": 0,
              }
            }
          >
            <View
              collapsable={false}
              forwardedRef={[Function]}
              onGestureHandlerEvent={[Function]}
              onGestureHandlerStateChange={[Function]}
              style={
                Object {
                  "flex": 1,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                    Object {
                      "translateX": 0,
                    },
                  ],
                }
              }
            >
              <View
                pointerEvents="none"
                style={
                  Object {
                    "backgroundColor": "rgb(242, 242, 242)",
                    "bottom": 0,
                    "left": 0,
                    "position": "absolute",
                    "shadowColor": "#000",
                    "shadowOffset": Object {
                      "height": 1,
                      "width": -1,
                    },
                    "shadowOpacity": 0.3,
                    "shadowRadius": 5,
                    "top": 0,
                    "width": 3,
                  }
                }
              />
              <View
                style={
                  Array [
                    Object {
                      "flex": 1,
                      "overflow": "hidden",
                    },
                    Array [
                      Object {
                        "backgroundColor": "rgb(242, 242, 242)",
                      },
                      undefined,
                    ],
                  ]
                }
              >
                <View
                  style={
                    Object {
                      "flex": 1,
                      "flexDirection": "column-reverse",
                    }
                  }
                >
                  <View
                    style={
                      Object {
                        "flex": 1,
                      }
                    }
                  >
                    <View
                      style={
                        Object {
                          "flex": 1,
                        }
                      }
                    >
                      <View
                        accessibilityIgnoresInvertColors={true}
                        style={
                          Object {
                            "alignItems": "flex-end",
                            "flex": 1,
                            "flexDirection": "row",
                            "justifyContent": "center",
                            "resizeMode": "cover",
                          }
                        }
                      >
                        <Image
                          source={
                            Object {
                              "process": [Function],
                            }
                          }
                          style={
                            Array [
                              Object {
                                "bottom": 0,
                                "left": 0,
                                "position": "absolute",
                                "right": 0,
                                "top": 0,
                              },
                              Object {
                                "height": undefined,
                                "width": undefined,
                              },
                              undefined,
                            ]
                          }
                        />
                        <View
                          style={
                            Object {
                              "alignItems": "center",
                              "marginBottom": 100,
                              "textAlign": "center",
                            }
                          }
                        >
                          <View
                            style={
                              Object {
                                "borderBottomColor": "#7b7b7b",
                                "borderBottomWidth": 0.5,
                                "borderStyle": "solid",
                                "marginVertical": 5,
                                "width": "100%",
                              }
                            }
                          >
                            <Text
                              style={
                                Object {
                                  "color": "#7b7b7b",
                                  "fontSize": 16,
                                  "left": 4,
                                  "position": "absolute",
                                  "top": 14,
                                }
                              }
                            >
                              Username
                            </Text>
                            <TextInput
                              allowFontScaling={true}
                              keyboardType="email-address"
                              onBlur={[Function]}
                              onChangeText={[Function]}
                              onFocus={[Function]}
                              rejectResponderTermination={true}
                              returnKeyType="next"
                              style={
                                Array [
                                  Object {
                                    "color": "#fff",
                                    "fontFamily": "Helvetica",
                                    "paddingBottom": 0,
                                  },
                                  Object {
                                    "width": "100%",
                                  },
                                ]
                              }
                              underlineColorAndroid="transparent"
                              value=""
                            />
                          </View>
                          <View
                            style={
                              Object {
                                "borderBottomColor": "#7b7b7b",
                                "borderBottomWidth": 0.5,
                                "borderStyle": "solid",
                                "marginVertical": 5,
                                "width": "100%",
                              }
                            }
                          >
                            <Text
                              style={
                                Object {
                                  "color": "#7b7b7b",
                                  "fontSize": 16,
                                  "left": 4,
                                  "position": "absolute",
                                  "top": 14,
                                }
                              }
                            >
                              Password
                            </Text>
                            <TextInput
                              allowFontScaling={true}
                              onBlur={[Function]}
                              onChangeText={[Function]}
                              onFocus={[Function]}
                              rejectResponderTermination={true}
                              secureTextEntry={true}
                              style={
                                Array [
                                  Object {
                                    "color": "#fff",
                                    "fontFamily": "Helvetica",
                                    "paddingBottom": 0,
                                  },
                                  Object {
                                    "width": "100%",
                                  },
                                ]
                              }
                              underlineColorAndroid="transparent"
                              value=""
                            />
                          </View>
                          <View
                            accessible={true}
                            focusable={true}
                            onClick={[Function]}
                            onResponderGrant={[Function]}
                            onResponderMove={[Function]}
                            onResponderRelease={[Function]}
                            onResponderTerminate={[Function]}
                            onResponderTerminationRequest={[Function]}
                            onStartShouldSetResponder={[Function]}
                            style={
                              Object {
                                "opacity": 1,
                              }
                            }
                          >
                            <View
                              style={
                                Array [
                                  Object {
                                    "alignItems": "center",
                                    "borderColor": "red",
                                    "height": 39.19,
                                    "justifyContent": "center",
                                    "marginVertical": 23.5,
                                    "width": 312,
                                  },
                                  Object {
                                    "opacity": 0.5,
                                  },
                                ]
                              }
                            >
                              <Image
                                source={
                                  Object {
                                    "process": [Function],
                                  }
                                }
                                style={
                                  Object {
                                    "borderRadius": 19,
                                  }
                                }
                              />
                              <View
                                style={
                                  Object {
                                    "alignItems": "center",
                                    "flex": 1,
                                    "justifyContent": "center",
                                    "position": "absolute",
                                  }
                                }
                              >
                                <Text
                                  style={
                                    Object {
                                      "alignItems": "center",
                                      "color": "#fff",
                                      "flex": 1,
                                      "fontWeight": "bold",
                                      "justifyContent": "center",
                                    }
                                  }
                                >
                                  LOGIN
                                </Text>
                              </View>
                            </View>
                          </View>
                        </View>
                      </View>
                    </View>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </View>
      </View>
    </View>
    <View
      pointerEvents="box-none"
      style={
        Object {
          "height": 64,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
          "zIndex": 1,
        }
      }
    />
  </RNCSafeAreaProvider>
</View>
`;
